helm/trivy/values.yaml

```diff
@@ -0,0 +1,77 @@
+nameOverride: ""
+fullnameOverride: ""
+
+image:
+  registry: docker.io
+  repository: aquasec/trivy
+  tag: 0.13.0
+  pullPolicy: IfNotPresent
+
+replicaCount: 1
+
+persistence:
+  enabled: true
+  storageClass: ""
+  accessMode: ReadWriteOnce
+  size: 5Gi
+
+resources:
+  requests:
+    cpu: 200m
+    memory: 512Mi
+  limits:
+    cpu: 1
+    memory: 1Gi
+
+rbac:
+  create: true
+  pspEnabled: true
+
+podSecurityContext:
+  runAsUser: 65534
+  runAsNonRoot: true
+  fsGroup: 65534
+
+securityContext:
+  privileged: false
+  readOnlyRootFilesystem: true
+
+trivy:
+  ## debugMode the flag to enable Trivy debug mode
+  debugMode: false
+  ## vulnType a comma-separated list of vulnerability types. Possible values are `os` and `library`.
+  vulnType: "os,library"
+  ## severity a comma-separated list of vulnerabilities severities to be displayed
+  severity: "UNKNOWN,LOW,MEDIUM,HIGH,CRITICAL"
+  ## ignoreUnfixed the flag to display only fixed vulnerabilities
+  ignoreUnfixed: false
+  ## skipUpdate the flag to enable or disable Trivy DB downloads from GitHub
+  ##
+  ## You might want to enable this flag in test or CI/CD environments to avoid GitHub rate limiting issues.
+  ## If the flag is enabled you have to manually download the `trivy.db` file and mount it in the
+  ## `/home/scanner/.cache/trivy/db/trivy.db` path (see `cacheDir`).
+  skipUpdate: false
```

Comment:
I think we're missing description of the `gitHubToken` property, which is referred to by the Secret.

Reply:
I think I missed to delete. In another comment mentioned the github token isn't needed to pull the database.

Reply:
Sorry for that @czunker You're right. I saw the Secret referring to the GH token and I mixed up things. Indeed we don't need the Secret and the GH token for Trivy client-server

Reply:
I overlooked this conversation, but I was talking about `insecure`, not `githubToken`. I meant `insecure` was unnecessary, but  `githubToken` is necessary for GitHub rate limiting.

Reply:
Fixed this in PR #769

Reply:
Absolutely. I mixed up pulling layers from registry server by client (does not require GH token) and downloading Trivy DB from GH which is subject to rate limit and we should be able to configure GH token.